test/AssemblySet_basic_test.py

```diff
@@ -6,98 +6,35 @@
 from typing import Any
 
 import pytest
-from SetAPI.assembly.AssemblySetInterfaceV1 import AssemblySetInterfaceV1
-from SetAPI.generic.constants import INC_ITEM_INFO, INC_ITEM_REF_PATHS, REF_PATH_TO_SET
+from SetAPI.generic.constants import (
+    ASSEMBLY,
+    ASSEMBLY_SET,
+    INC_ITEM_INFO,
+    INC_ITEM_REF_PATHS,
+    REF_PATH_TO_SET,
+)
 from SetAPI.SetAPIImpl import SetAPI
 
-API_CLASS = AssemblySetInterfaceV1
-SET_TYPE = "assembly"
-
-
-def save_assembly_set(
-    set_api_client: SetAPI,
-    context: dict[str, str | list],
-    ws_id: int,
-    set_name: str,
-    set_data: dict[str, Any],
-) -> dict[str, Any]:
-    """Given a set_name and set_data, save an assembly set."""
-    return set_api_client.save_assembly_set_v1(
-        context,
-        {
-            "workspace_id": ws_id,
-            "output_object_name": set_name,
-            "data": set_data,
-        },
-    )[0]
-
-
-@pytest.fixture(scope="module")
-def assembly_set(
-    assembly_refs: list[str],
-    set_api_client: SetAPI,
-    context: dict[str, str | list],
-    ws_id: int,
-) -> dict[str, int | str | dict[str, Any]]:
-    """A set with name, description, and items populated."""
-    set_name = "test_assembly_set"
-    set_description = "test_assemblies"
-    set_items = [{"label": "some_label", "ref": ref} for ref in assembly_refs]
-    set_data = {
-        "description": set_description,
-        "items": set_items,
-    }
-
-    result = save_assembly_set(set_api_client, context, ws_id, set_name, set_data)
-
-    return {
-        "obj": result,
-        "set_name": set_name,
-        "set_type": API_CLASS.set_type(),
-        "set_description": set_description,
-        "n_items": len(assembly_refs),
-        "second_set_item": set_items[1],
-    }
-
-
-@pytest.fixture(scope="module")
-def empty_assembly_set(
-    set_api_client: SetAPI,
-    context: dict[str, str | list],
-    ws_id: int,
-) -> dict[str, int | str | dict[str, Any]]:
-    """A set with no description and an empty items list."""
-    set_name = "empty_assembly_set"
-    # omit the set description and make the set items an empty list
-    set_data = {
-        "items": [],
-    }
-
-    result = save_assembly_set(set_api_client, context, ws_id, set_name, set_data)
-
-    return {
-        "obj": result,
-        "set_name": set_name,
-        "set_type": API_CLASS.set_type(),
-        # the class fills in the missing description field
-        "set_description": "",
-        "n_items": 0,
-    }
+SET_TYPE = ASSEMBLY_SET
+SET_ITEM_NAME = ASSEMBLY
 
 
+@pytest.mark.parametrize("ws_id", ["default_ws_id"], indirect=True)
```

Comment:
This is the start of some pytest magic. Pytest allows you to call fixtures indirectly, which means that you can put in a text string and it calls the fixture with that name. Probably easiest to skip ahead to the [conftest](https://github.com/kbaseapps/SetAPI/pull/82/files#diff-7dae78a00d8d69422a3bfbde96c7c5d794eede504a0df763853896a1ba1b5ff2) file to see the changes there first.